src/main/steps/applicant1/hub-page/right-column/content.ts

```diff
@@ -4,29 +4,42 @@ import { CommonContent } from '../../../common/common.content';
 import { APPLICANT_2, CHECK_CONTACT_DETAILS, RESPONDENT } from '../../../urls';
 
 const en = ({ isDivorce, isApplicant2, userCase }: CommonContent) => ({
-  applicationDownloadLink: `<a class="govuk-link" href="/downloads/${
-    isDivorce ? 'divorce-application' : 'application-to-end-civil-partnership'
-  }"
-  download="${isDivorce ? 'Divorce-application' : 'Civil-partnership-application'}">View the ${
-    isDivorce ? 'divorce application' : 'application to end your civil partnership'
-  } (PDF)</a>`,
-  certificateOfServiceDownloadLink:
-    '<a class="govuk-link" href="/downloads/certificate-of-service" download="Certificate-of-service">View your ‘certificate of service’ (PDF)</a>',
-  respondentAnswersDownloadLink: `<a class="govuk-link" href="/downloads/respondent-answers"
-  download="Respondent-answers">View the response to the ${
-    isDivorce ? 'divorce application' : 'application to end your civil partnership'
-  } (PDF)</a>`,
-  deemedOrDispensedDownloadLink: `View the court order granting your application for
-  ${
-    userCase.alternativeServiceOutcomes?.[0].value.alternativeServiceType === AlternativeServiceType.DISPENSED
-      ? 'dispensed'
-      : 'deemed'
-  } service (PDF)`,
-  deemedOrDispensedDownloadReference: `/downloads/${
-    userCase.alternativeServiceOutcomes?.[0].value.alternativeServiceType === AlternativeServiceType.DISPENSED
-      ? 'certificate-of-dispense-with-service'
-      : 'certificate-of-deemed-as-service'
-  }`,
+  applicationDownload: {
+    name: 'Divorce-Application',
+    reference: `/downloads/${isDivorce ? 'divorce-application' : 'application-to-end-civil-partnership'}`,
+    link: `View the ${isDivorce ? 'divorce application' : 'application to end your civil partnership'} (PDF)`,
```

Comment:
maybe this should be called text and reference should be called link? im happy for reference to stay but link is more like text.

Reply:
kinda funny how name is more like reference (of the download file) and reference is more like link and link is more like text. if this makes sense.

Reply:
yeah you're right, I'll update those